client/src/App.js

```diff
@@ -10,8 +11,19 @@ import Dashboard from "./pages/Dashboard";
 import "./App.css";
 
 function App() {
-  const [loggedIn, setLoggedIn] = React.useState(localStorage.getItem("user"));
-
+  // const [loggedIn, setLoggedIn] = React.useState(localStorage.getItem("user"));
+  const getUser = async () => {
+    try {
+      const {data} = await axios.get('/users/auth');
+      localStorage.setItem("userInfo",JSON.stringify(data.user));
```

Comment:
So we don't actually want to use localStorage at all, since we're using cookies - everything should be coming from the back end